recipes/pycgns/bld.bat

```diff
@@ -0,0 +1,9 @@
+
+"%PYTHON%" -c "cmd=\"with open('setup.py', 'r') as f:\n  lines= f.readlines()\nwith open('setup.py', 'w') as f:\n  for line in lines:\n    f.write(line.replace('PTHREAD\\': 1', 'PTHREAD\\': 0'))\n\"; exec(cmd)"
```

Comment:
This is fine, but a source patch might be clearer. See the conda-build documentation.

Reply:
The source patch was just done for the 4.3.1 release so it can be removed. I just let it there since it works also with older versions.